File: 프로그래머스/0/120846. 합성수 찾기/합성수 찾기.js
function solution(n) {
    let answer = 0

    function find(x) {
        let arr = [];
        for(let i=1;i<=x;i++) {
            if(x%i===0) {
                arr.push(i)
            }
        }
        if(arr.length>=3) {
            answer+=1
        }
    }
        for(let i=1;i<=n;i++) {
        find(i)
    }
    return answer
}

File: 프로그래머스/0/120846. 합성수 찾기/합성수 찾기.java
public class Solution {
    public int solution(int n) {
        int answer = 0;

        for (int i = 1; i <= n; i++) {
            if (find(i)) {
                answer++;
            }
        }
        return answer;
    }

    private boolean find(int x) {
        int count = 0;
        for (int i = 1; i <= x; i++) {
            if (x % i == 0) {
                count++;
            }
            if (count >= 3) {
                return true;
            }
        }
        return false;
    }


}
